distribution/docker/src/docker/Dockerfile.wolfi

```diff
@@ -0,0 +1,177 @@
+################################################################################
+# This Dockerfile was generated from the template at distribution/src/docker/Dockerfile
+#
+# Beginning of multi stage Dockerfile
+################################################################################
+
+################################################################################
+# Build stage 1 `builder`:
+# Extract Elasticsearch artifact
+################################################################################
+
+FROM docker.elastic.co/wolfi/chainguard-base:latest AS builder
+
+# Install required packages to extract the Elasticsearch distribution
+
+RUN for iter in 1 2 3 4 5 6 7 8 9 10; do \
+      apk update && DEBIAN_FRONTEND=noninteractive apk add curl  && \
+      exit_code=0 && break || \
+        exit_code=\$? && echo "apk error: retry $iter in 10s" && sleep 10; \
+    done; \
+    exit $exit_code
+
+# `tini` is a tiny but valid init for containers. This is used to cleanly
+# control how ES and any child processes are shut down.
+#
+# The tini GitHub page gives instructions for verifying the binary using
+# gpg, but the keyservers are slow to return the key and this can fail the
+# build. Instead, we check the binary against the published checksum.
+RUN set -eux ; \
```

Comment:
tini is the one package we install that's not from the distribution repositories. This leaves us open to CVEs here (in theory).